src/pretix/control/templates/pretixcontrol/event/mail.html

```diff
@@ -17,6 +17,8 @@ <h1>{% trans "E-mail settings" %}</h1>
                 {% bootstrap_field form.mail_text_signature layout="control" %}
                 {% bootstrap_field form.mail_bcc layout="control" %}
                 {% bootstrap_field form.mail_attach_ical layout="control" %}
+                {% bootstrap_field form.mail_sales_channel_placed_paid layout="control" %}
+                {% bootstrap_field form.mail_sales_channel_ticket_reminder layout="control" %}
```

Comment:
Maybe in another place? It has sensible defaults and most users don't need it/don't even need to understand what sales channels are...

Reply:
For the ticket reminder, I'd put it in the same box as the ticket reminder text, because that's also where you activate the reminder in the first place, so the context is pretty clear.

For the placed/paid/free, this doesn't work since it would need to go in 3 boxes, so maybe the place you chose is a good one for that, even though that makes it a little inconsistent.